content/influxdb/v2.0/reference/flux/stdlib/contrib/zenoss/event.md

```diff
@@ -0,0 +1,172 @@
+---
+title: zenoss.event() function
+description: >
+  The `zenoss.event()` function sends an event to Zenoss.
+menu:
+  influxdb_2_0_ref:
+    name: zenoss.event
+    parent: Zenoss
+weight: 202
+---
+
+The `zenoss.event()` function sends an event to [Zenoss](https://www.zenoss.com/).
+
+```js
+import "contrib/bonitoo-io/zenoss"
+
+zenoss.event(
+  url: "https://example.zenoss.io:8080/zport/dmd/evconsole_router",
+  username: "example-user",
+  password: "example-password",
+  action: "EventsRouter",
+  method: "add_event",
+  type: "rpc",
+  tid: 1,
+  summary: "",
+  device: "",
+  component: "",
+  severity: "Critical",
+  eventClass: "",
+  eventClassKey: "",
+  collector: "",
+  message: ""
+)
+```
+
+## Parameters
+
+### url
+({{< req >}})
+Zenoss events [public endpoint](https://help.zenoss.com/zsd/RM/configuring-resource-manager/enabling-access-to-browser-interfaces/creating-and-changing-public-endpoints) URL.
```

Comment:
Like in `endpoint()` function, "Zenoss router endpoint URL" seems more correct to me.